tests/integration/components/paper/check-box-test.js

```diff
@@ -0,0 +1,40 @@
+import { module, test }        from 'qunit';
+import { render, find, click } from '@ember/test-helpers';
+import hbs                     from 'htmlbars-inline-precompile';
+import { setupRenderingTest }  from 'ember-qunit';
+
+module('Integration | Component | {{paper/check-box}}', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test(`renders a checkbox with class check-box`, async function(assert) {
+    await render(hbs`{{paper/check-box}}`);
+    assert.ok(find('.md-checkbox'));
+  });
+
+  test(`has given value`, async function(assert) {
+    this.set('selected', true);
+    await render(hbs`{{paper/check-box value=selected}}`);
+    assert.ok(find('.md-checkbox.md-checked'));
+
+    this.set('selected', false);
+    assert.notOk(find('.md-checkbox.md-checked'));
```

Comment:
It would be more readable to have something like `hasClass('md-checked')`, so you're sure the checkbox itself is still present